src/ordinal.rs

```diff
@@ -107,9 +107,17 @@ impl Ordinal {
 
     let cycle_start_epoch = cycle_number * CYCLE_EPOCHS;
 
-    let cycle_progression = period_offset
```

Comment:
`cycle_progression` should be a number between 0 and 1_260_00 (6*210_00) that represents the amount of blocks passed in this cycle.